docs/rules/no-get-by-assert.md

```diff
@@ -0,0 +1,51 @@
+# Disallow `getBy*` queries as assertion method itself (no-get-by-assert)
```

Comment:
The rule name sounds a bit strange to me. When it comes to preferences, ESLint rules are usually named `prefer-...`, I think it should be the case here.
`no-get-by-assert` implies here that doing assertions with `getBy` queries is wrong. I think something like `prefer-query-by-assert` is more clear to the user as discussed in the [issue](https://github.com/Belco90/eslint-plugin-testing-library/issues/28)

Reply:
I'm happy to rename this to `prefer-*` rule but I would avoid using query-by for same reason I commented before, so I would go for something like `prefer-explicit-assert`. What do you think?

Reply:
`prefer-explicit-assert` sounds good to me!